repository/repository.go

```diff
@@ -122,17 +128,17 @@ func (r *Repository) LoadBlob(t pack.BlobType, id backend.ID) ([]byte, error) {
 	}
 
 	// decrypt
-	plain, err := r.Decrypt(buf)
+	buf, err = r.decryptTo(buf, cbuf)
```

Comment:
and `plaintextBuf` for consistency then? I think that would make this code a bit easier to follow